marathon-bigip-ctlr.py

```diff
@@ -745,7 +909,7 @@ def __init__(self, marathon, verify_interval, bigip):
     def do_reset(self):
         """Process the Marathon state and reconfigure the BIG-IP."""
         with self.__condition:
-            while True:
+            while True:  # pylint: disable=too-many-nested-blocks
```

Comment:
Any chance that we can actually just make the linter happy?  (As a human I also kinda feel this is a lot of nested blocks too)